components/brave_wallet_ui/page/reducers/page_reducer.ts

```diff
@@ -233,6 +235,10 @@ export const createPageSlice = (initialState: PageState = defaultState) => {
 
       updateLocalIpfsNodeStatus (state, { payload }: PayloadAction<boolean>) {
         state.isLocalIpfsNodeRunning = payload
+      },
+
+      setIsHardwareOnboarding (state, { payload }: PayloadAction<boolean>) {
```

Comment:
I think we can accomplish this without adding new redux state by adding routes for hardware-onboarding